rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  belongs_to :person
+  belongs_to :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] You need ensure that rental is adding itself to ``` @book and @person ``` in order to achieve the belongs-to relationship. For instance:
```ruby
def initialize(date, book, person)
    .
    .

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
```